cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -2098,22 +2129,21 @@ Slice::Python::CodeVisitor::writeDocstring(const optional<DocComment>& comment,
         _out << nl << "Enumerators:";
         for (const auto& enumerator : enumerators)
         {
-            _out << nl << enumerator->mappedName() << " -- ";
+            _out << nl << nl << "- " << enumerator->mappedName();
```

Comment:
I assume the extra nl is intentional?

Reply:
Yep, the first `nl` moves to the next line, and the second one adds a blank line between 2 enumerators.